docs/decisions/0007-sys-path-modification-removal.rst

```diff
@@ -32,11 +34,179 @@ This deprecation will take place in the following steps:
 
 4. Fix all instances where the ``sys.path``-based modules were ``patch``-ed in unit tests, as those patches no longer work.
 
-5. Once all instances of the ``sys.path``-based imports have been migrated in the ``edx-platform`` codebase, officially deprecate that import pattern for the next open-source release (as per the standard deprecation process).
```

Comment:
We did not go through the DEPR process, since this was purely a (breaking) refactor and not a feature change.